src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +271,89 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool IsAutoAccessorList(SyntaxToken token)
+        {
+            if(token.Parent != null && token.Parent is AccessorListSyntax accessorList)
+            {
+                return accessorList.Accessors.All(a => a.Body == null);
+            }
+
+            return false;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+        {
+            if(node != null && node.Parent != null && node.Parent is AccessorListSyntax accessorList)
+            {
+                return accessorList.Accessors.All(a => a.Body == null);
+            }
+
+            return false;
+        }
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxToken token)
+        {
+            if(token.Parent != null && token.Parent is AccessorListSyntax accessorList)
+            {
+                var declaration = accessorList.Parent;
+                if (declaration == null)
+                {
+                    return false;
+                }
+
+                var lastChild = declaration.ChildNodes().LastOrDefault();
+                if(lastChild == null || lastChild == accessorList)
+                {
+                    return false;
+                }
+                //var children = declaration.ChildNodes().ToArray();
+                //var accessorListIndex = children.IndexOf(accessorList);
+                //if (accessorListIndex == -1 || accessorListIndex == children.Length - 1)
+                //{
+                //    return false;
+                //}
+
+                return true;
+            }
+
+            return false;
+        }
+
+        private static int LineBreaksBeforeOpenBrace(SyntaxToken currentToken, SyntaxToken nextToken)
+        {
+            if (nextToken.Parent.IsKind(SyntaxKind.Interpolation) ||
+                nextToken.Parent is InitializerExpressionSyntax ||
+                IsAutoAccessorList(currentToken) ||
+                IsAutoAccessorList(nextToken))
```

Comment:
it's not really clear to me why you're checking both current token and next token.  seems confusing.  i would prefer far more explicit checks of what is going on for exact scenarios.  or, if both checks are needed, having a comment explaining each one.

Reply:
When debugging I could see them both:
- current = variable name, next = open brace
- current = semicolon (before the get), next =open brace
Do you want me to put comments in the code?

Reply:
yes.  and please break this out into separate checks.  addign comments in the middle of the expression will not be fun to read/maintain :)

try to wrap aronud 120 columns for comments.  thanks!

Reply:
Okay.
FYI the two methods `LineBreaksBeforeOpenBrace` and `LineBreaksBeforeCloseBrace` are currently identical.
IMO it is good for clarity reasons because it allows to support more cases in the future.
If you want me to merge them, please let me know.

Reply:
I noticed that one case was not used anymore as it is already covered in another method.
I totally removed the ifs that are not used and left the two methods (which are now slightly different)